### ISSUE TITLE
Content Suggestion: NAME OF CONTENT TO BE ADDED

### ISSUE THREAD

**Issue by wiz** [bitcoin-only/bitcoin-only#415]
1) Link to Content: https://liquiditi.io/
2) Briefly Describe Content: No-KYC swap service between BTC and L-BTC, L-USDT, L-CAD
3) Suggested Section: Bitcoin Layer 2 Network swap services

**Comment by 6102bitcoin** [comment]
What page are you imagining for this? A dedicated layer 2 section?

**Comment by wiz** [comment]
Hmm, not sure. Are there any no-KYC swap services doing USD to BTC currently?

**Comment by 6102bitcoin** [comment]
bisq?

**Comment by 6102bitcoin** [comment]
Still unsure where to add this. I think we need a Layer 2 page

**Comment by pardus79** [comment]
If it's a service that gets paid a fee to move funds between sidechains, could it go on "Spend Bitcoin"?

**Comment by wiz** [comment]
Yeah, IMO a Layer 2 page makes sense now that fee market is developing we will see more Layer 2 stuff going forward.
